client/chainlink_models.go

```diff
@@ -585,8 +586,16 @@ observationSource = """
 	return marshallTemplate(w, "Webhook Job", webHookTemplateString)
 }
 
-func ObservationSourceSpec(url string) string {
+// ObservationSourceSpecHttp creates a http GET task spec for json data
+func ObservationSourceSpecHttp(url string) string {
```

Comment:
I think this is a Go thing, but with acronyms you typically capitalise them all. So `HTTP`, `URL` etc.